src/config/config.ts

```diff
@@ -91,6 +92,7 @@ export type PickerConfigInternal = {
   getEmojiUrl: GetEmojiUrl;
   searchDisabled: boolean;
   skinTonePickerLocation: SkinTonePickerLocation;
+  unicodeToHide: string[];
```

Comment:
Since this is fully internal, we can use a `Set<string>`, this will make the lookup faster, and when we're talking about thousands of emoji renders - it makes an impact.